src/weekly-contributions.ts

```diff
@@ -5,10 +5,13 @@ import {inspect} from 'util'
 type IssueStates = 'OPEN' | 'CLOSED'
 type PRStates = IssueStates | 'MERGED'
 
+const loginToNameMap: {[key: string]: string} = {}
+
 async function main() {
   const token = core.getInput('github-token')
   const dateStr = core.getInput('date')
   const markdownPrefix = core.getInput('markdown-prefix') || ''
+  const loginToNameMap = core.getInput('login-to-name-map') || ''
```

Comment:
```suggestion
  loginToNameMap = JSON.parse(core.getInput('login-to-name-map')) || {}
```
remove const to reassign existing variable. Also github action inputs can only be strings, so the easiest way to get what we're after here is to use JSON and parse it on it's way it.
When adding the input in the github action, will have to be proper JSON with full `"` everywhere
```yml
login-to-name-map: |
  {
    "Irev-Dev": "Kurt",
    "hanbollar": "Hannah",
    "iterion": "Adam",
    "JBEmbedded": "JB",
    "jessfraz": "Jess",
    "jgomez720": "Josh",
    "JordanNoone": "Jordan",
    "mansoorsiddiqui": "Mansoor",
    "vonniwilliams": "Vonni"
  }
```

Reply:
√